daemon/cmd/kube_proxy_replacement.go

```diff
@@ -216,19 +219,22 @@ func initKubeProxyReplacementOptions() (bool, error) {
 	}
 
 	if option.Config.EnableNodePort {
-		found := false
-		if h := probesManager.GetHelpers("sched_act"); h != nil {
-			if _, ok := h["bpf_fib_lookup"]; ok {
-				found = true
+		if probesManager != nil {
```

Comment:
nit: just checking if we're not on dry mode here might make this easier for someone to read.

Reply:
Fixed.